benchadapt/python/benchadapt/result.py

```diff
@@ -68,6 +76,10 @@ class BenchmarkResult:
     github: Dict[str, Any] = field(default_factory=github_info)
     error: str = None
 
+    def __post_init__(self) -> None:
+        if not self.run_name and self.github.get("commit"):
+            self.run_name = f"{self.run_reason}: {self.github['commit']}"
```

Comment:
We might not want to default to this until we clean up the way we identify the main branch for history purposes. IIUC, this would mean that unless someone configures `run_name`, if they are on a PR branch, they would get `commit: ...` as the default, yeah?

Reply:
They'd get whatever is specified in `run_reason`, which has no default (and therefore could be `None`, which is a bit gross). I don't currently have a way to autodetect what that should be, as adapters could be run in so many ways; populating `run_reason` is still something users will have to do. We could make them specify `run_name` as well, but I suspect most will do it poorly.

Reply:
Aah ah ah right. I saw too many "commit" around there and transposed that to the second line, my bad